e2e/framework/addon/eso.go

```diff
@@ -17,7 +17,7 @@ import (
 	"os"
 
 	// nolint
-	. "github.com/onsi/ginkgo/v2"
+	ginkgo "github.com/onsi/ginkgo/v2"
```

Comment:
We can keep the dot import here I think. Makes it simpler with e2e tests since we use a bunch of ginkgo stuff.

Or, if we start specifying imports, we should do everywhere (in another PR)

Reply:
Done!